src/components/layout.js

```diff
@@ -1,14 +1,27 @@
 import Logo from '../assets/logo.png'
 import React from 'react'
 import { Routes } from './routes'
+import { Link } from 'react-router-dom';
 const NavBar = () => (
   <nav
     className="navbar navbar-expand-lg navbar"
-    style={{ backgroundColor: '#243387' }}
+    style={{ backgroundColor: '#243387', color: "#FFFFFF" }}
   >
     <a className="navbar-brand" href="#">
       <img src={Logo} height="50" alt="" />
     </a>
+    <div className="collapse navbar-collapse pull-right" id="navbarSupportedContent">
+      <ul className="navbar-nav mr-auto">
+        {(
+          window.location.href.includes("main")) ? (<li className="nav-item ">
```

Comment:
Checking by `main` string is a brittle solution. What if the route path changes? 
https://github.com/pamply/luxoft-react-app/pull/4/files#diff-6bda2621f1877130b3d7eeb6e0b0da35L22

Reply:
i will rewrite it, to have some global variable o what do you suggest?